Makefile

```diff
@@ -68,12 +68,12 @@ install: ## Make a conda env with dist/*.whl and dist/*.tar.gz installed
 	conda env remove -y -n $(ENV)-install
 
 bdist: ## Make a dist/*.whl binary distribution
-	$(SA) $(ENV) && python setup.py bdist_wheel $(POST_SDIST) \
-		&& rm -rf *.egg-info
+	$(SA) $(ENV) && python setup.py bdist_wheel $(POST_SDIST) -r pypi \
```

Comment:
The `-r pypi` argument should be part of the `$(POST_SDIST)` variable since it only applies to upload and should only be used on `make release` invocations.  Same goes for the `sdist:` target on L75.

If I run `python setup.py bdist_wheel -r pypi` I get the following output:
```
python setup.py bdist_wheel -r pypi
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option -r not recognized
```

Reply:
yes it makes sense to specify the arguments for pypi, already pushed the changes